microsite/data/plugins/scaffolder-backend-dotnet.yaml

```diff
@@ -0,0 +1,9 @@
+---
+title: Scaffolder Backend .NET
+author: Alef Carlos
+authorUrl: https://github.com/alefcarlos
+category: Scaffolder
+description: Here you can find all .NET related actions to improve your scaffolder.
+documentation: https://github.com/alefcarlos/plusultra-dotnet-backstage-plugins/blob/main/plugins/scaffolder-dotnet-backend/README.md
+iconUrl: img/scaffolder-backend-dotnet-icon.png
+npmPackageName: '@plusultra/plugin-scaffolder-dotnet-backend'
```

Comment:
You'll need a new line here at the end. The builds when they run will fail on this